docker-compose.yml

```diff
@@ -30,6 +30,11 @@ services:
       && python manage.py loaddata urgent_alerts.yaml
       && python manage.py loaddata land_use_ti.yaml
       && python manage.py loaddata land_use_classes.yaml
+      && python manage.py loaddata acao_cadastrada.yaml
```

Comment:
Is it about documental? Please add `documental_` as prefix?

Reply:
Modified.